src/views/open-api-docs-view/components/operation-details/index.tsx

```diff
@@ -3,19 +3,70 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import { DevCodeBlock } from '../dev-code-block'
-import type { OperationProps } from 'views/open-api-docs-view/types'
+import { PropertyDetails, PropertyDetailsProps } from '../property-details'
+import s from './operation-details.module.css'
 
 /**
- * TODO: implement this presentation component.
+ * Render request and response data for an operation.
  */
-export function OperationDetails({ operation }: { operation: OperationProps }) {
+export function OperationDetails({
+	requestData,
+	responseData,
+}: {
+	requestData: { heading: string; propertyDetails: PropertyDetailsProps[] }[]
+	responseData: { heading: string; propertyDetails: PropertyDetailsProps[] }[]
+}) {
```

Comment:
⛏️ Since its such a long line length, thoughts on defining this in an interface above?